mixins/form.jsx

```diff
@@ -96,7 +95,7 @@ module.exports = {
     this.updateHeight();
   },
   submit: function(action, props, callback) {
-    props.localeCode = this.state.localeCode || "US";
+    props.locale = this.props.locales[0];
```

Comment:
We now get this from client.jsx. Feels more consistent to refer to locales in one way all over and define it as high level as we can.